developer-docs-site/docs/releases/cli-release.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "CLI Release"
+slug: "cli-release"
+---
+
+## Aptos CLI Releases
+
+### How to install
+
+If you have already installed the CLI:
+* With brew: `brew upgrade aptos`
+* With the prepackaged CLI: `aptos upgrade`
+
+[Read about different ways to install Aptos CLI here!](https://aptos.dev/tools/install-cli/)
+
+Managed install:
+* macOS: [Install with brew](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/homebrew/README.md)
+* Linux / Windows: [install with a python script for other platforms](https://aptos.dev/tools/install-cli/automated-install)
+* [Build from source](https://aptos.dev/tools/install-cli/build-from-source)
```

Comment:
Let's not mention this here, linking to the installation in aptos.dev is sufficient. Otherwise we have to maintain two different places.